Mathlib/CategoryTheory/Abelian/GrothendieckAxioms.lean

```diff
@@ -54,58 +56,86 @@ universe v v' u u' w
 
 variable (C : Type u) [Category.{v} C]
 
-attribute [local instance] hasCoproducts_of_finite_and_filtered
+/--
+A category `C` which has colimits of shape `J` is said to have `AB` of shape `J` provided that
+colimits of shape `J` are exact.
+-/
+class ABOfShape (J : Type u') [Category.{v'} J] (C : Type u) [Category.{v} C]
```

Comment:
I would prefer a more specific name, like `HasExactColimitsOfShape`, and following #19206, this can be a prop.